tests/integration/test_hitl_loop_minimal.py

```
@@ -0,0 +1,69 @@
+    Absolute minimal test: just verify HITL in loop can pause and resume.
+    """
+    # Single agent step followed by HITL
+    step1 = Step(name="work", agent=StubAgent(["done"]))
+    step2 = HumanInTheLoopStep(name="hitl", message_for_user="OK?")
+
+    # Loop that exits after first iteration
+    # NOTE: Can't use >> operator with HITL, must use Pipeline(steps=[...])
+    loop = LoopStep(
+        name="simple_loop",
+        loop_body_pipeline=Pipeline(steps=[step1, step2]),
+        exit_condition_callable=lambda output, context: True,  # Always exit after first iteration
```

Ruff (0.13.3)

Unused lambda argument: `output`

(ARG005)